plugins/generator-1.14.4/forge-1.14.4/procedures/world_use_bonemeal.java.ftl

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please base the logic from BoneMealItem#onItemUse(ItemUseContext). Current logic does not reflect the way it works in vanilla, OR-ing seagrass and bonemeal is not how vanilla does it.

Reply:
Isn't it how it's done by `IDispenseBehavior#init` when registering the bonemeal dispenser behavior? It doesn't evaluate the seagrass if the bonemeal is successful

Reply:
I was checking BoneMealItem#onItemUse(ItemUseContext), where impl. is a bit more complex, but did not check this one. You are correct ;)